### PR TITLE
Fixed inviting deleted user bug

### PR DESCRIPTION
**Pull request by Aleks4ndr**
I've fixed 'Invite deleted user bug' but maybe it is not right solution

### CODE REVIEW

File: app/services/batch_invite_job.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ def initialize(params, current_user)
 
   def run
     tell_devise_if_okay_to_resend_invitations
+    complitely_delete_deleted_users
```
Comment (tansaku) [inline app/services/batch_invite_job.rb]: "completely_delete_deleted_users"

or

"purge_deleted_users"

File: spec/services/batch_invite_job_spec.rb [hunk 2]
```diff
@@ -49,6 +49,13 @@
          (org.id+1) => 'Error: Email has already been taken'}
       )
     end
+
+
+    it 'resend invitation to a deleted user' do
+      deleted_user = FactoryGirl.create(:user,email: 'YES@hello.com', deleted_at: "2015-11-11 08:37:14")
+      expect(-> { subject }).to change(User, :count).by(1)
```
Comment (tansaku) [inline spec/services/batch_invite_job_spec.rb]: should re-run the invitation given that the delete took place, and make sure that the error doesn't occur

File: app/services/batch_invite_job.rb [hunk 3]
```diff
@@ -37,4 +38,9 @@ def result_of_inviting user
       'Invited!'
     end
   end
+
+  def complitely_delete_deleted_users
+    User.deleted.delete User.deleted.where(email: @invites.flat_map{|org_id, email| email.downcase}).pluck(:id)
```
Comment (johnnymo87) [inline app/services/batch_invite_job.rb]: Why not use the `really_delete` method that [the docs talk about](https://github.com/rubysherpas/paranoia)?

**Comment by tansaku** [comment]
@Aleks4ndr BTW, here's the video of our chat from Monday in case you want to review:

https://www.youtube.com/watch?v=elmG4FIGY1I

**Pull request closed without merging**